Article/Article.js

```diff
@@ -1,26 +1,30 @@
-// Start at `let articles`
+// Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(element) {
-    // assign this.element to the passed in article element
-    this.element;
+  constructor(domElement) {
+    // assign this.domElement to the passed in domElement
+    this.domElement = domElement
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector(".expandButton");
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference (or article element), calling the expandArticle method.
-    
+    this.expandButton.innerText= "expand";
+    // Set a click handler on the expandButton reference, calling the expandArticle method.
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
```

Comment:
Another way to do this would be:
`this.expandButton.addEventListener('click', () => { this.expandArticle() });`
That way you don't need to use the bind keyword 😄